util/process.go

```diff
@@ -25,14 +25,21 @@ import (
 )
 
 // StartHTTPServer starts an HTTP server on the given address.
-func StartHTTPServer(addr string) error {
+func StartHTTPServer(addr, certFile, keyFile string) error {
 	sock, err := net.Listen("tcp", addr)
 	if err != nil {
 		return err
 	}
 	go func() {
 		glog.Info("HTTP server starting")
-		http.Serve(sock, nil)
+		if certFile != "" || keyFile != "" {
```

Comment:
Can this work without both bring present, or are you just wanting to interpret either flag being non-empty as operator intent to run TLS but let `http.ServeTLS` 
 handle this potential error case? (maybe add a comment to that effect if that's what you're after)

Reply:
Yes, that was my intention. Added a comment.